grafana/Dockerfile

```
@@ -29,7 +29,7 @@ RUN \
     && if [ "${BUILD_ARCH}" = "amd64" ]; then \
         apt-get install -y --no-install-recommends \
             libasound2=1.2.4-1.1 \
-            libcups2=2.3.3op2-3+deb11u2 \
+            libcups2=2.3.3op2-3+deb11u2  \
```

Unrelated change to this PR

```suggestion
            libcups2=2.3.3op2-3+deb11u2 \
```